src/frontend/components/router-tree/router-tree.ts

```diff
@@ -52,7 +52,7 @@ export class RouterTree {
     };
 
     const nodes = this.tree(d3.hierarchy(
-      root.children.length > 1 ? root : root.children[0], d => d.children));
+      root.children.length === 0 || root.children.length > 1 ? root : root.children[0], d => d.children));
```

Comment:
Please add round brackets, no everyone is hip to operator precedence, plus makes the code readable to more people.

I am assuming root.children[0] is a boolean value?

Other than this LGTM.